src/components/ProgramCertificatesList/test/ProgramCertificatesList.test.jsx

```diff
@@ -22,7 +22,7 @@ describe('program-certificates-list', () => {
   it('renders the Program Certificates List', () => {
     render(<ProgramCertificatesList />);
 
-    expect(screen.getByText('Verifiable Credentials')).toBeTruthy();
+    expect(screen.queryAllByText('Verifiable Credentials')).toBeTruthy();
```

Comment:
these tests were failing locally because there were 2 instances in the DOM where 'Verifiable Credentials' existed, same with 'My Learner Records' in the following test file.

Reply:
what's different about this branch that made those tests pass before and not here? Or had they always been failing locally?

Reply:
###TLDR; 
A child component of these tested components was checking ENABLE_VERIFIABLE_CREDENTIALS in the ConfigDocument, which returned `true` due to the `env.config.js` existing in my branch. This led to the `screen` seeing `My Learner Records` and `Verifiable Credentials` twice, as noted in a screenshot below.
 
Even though the variables are an exact copy of `.env.test`, because `process.env` doesn't get initialized in these tests, the child component's check for ENABLE_VERIFIABLE_CREDENTIALS would be `undefined`. 

###Solution: 
Mocking the child component (`NavigationBar`) in the tests, as I think it originally should have been.

###Longer Read:
I found it! Alright, so in the `.env.development` and `.env.test` files, the variable ENABLE_VERIFIABLE_CREDENTIALS is set to 'true'. 

When set to true, it will show something like the screenshot below: 
<img width="671" alt="Screenshot 2024-02-07 at 1 54 56 PM" src="https://github.com/openedx/frontend-app-learner-record/assets/62807795/31a13d70-d5a0-480b-bdc3-4a0df4b97615">

But because the components are tested separately in Jest and the components -- and their children -- don't reference `process.env.ENABLE_VERIFIABLE_CREDENTIALS`, they can safely ignore the `.env.development` and `.env.test` boolean for ENABLE_VERIFIABLE_CREDENTIALS and will instead expect something like the following screenshot: 
<img width="854" alt="Screenshot 2024-02-07 at 1 54 51 PM" src="https://github.com/openedx/frontend-app-learner-record/assets/62807795/cefe6122-38ea-4ad9-9e73-c7a2be2bcab2">
However, I did see that there's[ a child component within these tests that calls](https://github.com/openedx/frontend-app-learner-record/blob/7abddcce2852cf8bf293722ba54cc89e83b5c9ab/src/components/NavigationBar/NavigationBar.jsx#L19) `getConfig().ENABLE_VERIFIABLE_CREDENTIALS`. This will return `undefined` because, again, the `.env.*` variables weren't initialized into the ConfigDocument as the component being tested doesn't go through the `initialize` process that occurs in the parent App. 

Now, the reason why these tests had to be changed was because I have been using a `env.config.js` file in my repo, which is copy/paste of the values in `env.development`. The _problem_ with that is the child component in these tests that calls `getConfig().ENABLE_VERIFIABLE_CREDENTIALS` will now pick up on the fact that ENABLE_VERIFIABLE_CREDENTIAL is `true` because the `env.config.js` has been merged into the config via the `setupTest.js` file [that I modified](https://github.com/openedx/frontend-app-learner-record/pull/275/files#diff-5b3172badbd948b02cae24fd87fba012dc42328eff3609ded67ded7f780cd196).